gaj23/get-going-gone
# Gaj/feature/deployment

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

This branch contains the necessary information to bundle it together.


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.

NA

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please see `dist` folder for webpack compiling.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Check if deployment link is functional. 

### Steps To Take
> Any questions and possible steps moving forward.

Edit README steps for viewing & contributing.

#### Checklist:
